app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,38 @@
+class ExpensesController < ApplicationController
+  
+  def create
+    begin
+      expense = Expense.create(expense_params)
```

Comment:
Replace this with an actual `params.require` call. We should probably require everything other than `comments`, `due_date`. Remember to get the `hideout_id` from `params[:payload][:hideout_id]`.

Reply:
Why is this required? I thought the hideout_id will be specified in params[:expenses], or am I missing something?